### DISCUSSION TITLE
[ARCHIVED] Продумать путь адаптации старых проектов к нашей методологии

### DISCUSSION THREAD

**Discussion by azinit** [feature-sliced/documentation#22] [Archive]
Т.е. условно:  "Есть большой монолит"

Шаг 1.  Разбиваем на самостоятельные блоки
Шаг 2.  Сводим связность между блоками к минимуму
и т.д.

**Comment by sergeysova** [comment]
Кажется что нужно начать с низкоуровневых блоков, вроде shared, затем постепенно переходить к более высокоуровневым абстракциям -> entities -> features -> pages -> processes

**Comment by azinit** [comment]
Мб также стоит в целом применить разделение монолита по схему `1. by domains => 2.by types` 

https://github.com/feature-sliced/wiki/discussions/31#discussioncomment-451507
